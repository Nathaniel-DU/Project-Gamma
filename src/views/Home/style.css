.mint-button {
    width: 90%;
    background-color: #ADEFD1FF;
    border: black 1px solid;
    /* padding: 30px; */
    border-radius: 15px;
}

.header {
    margin-bottom: 10rem;
}

.staysafe-logo {
    display: block;
    margin: auto;
}

main {
    height: 100%;
}

h1.header {
    margin-top: 1.5rem;
}
