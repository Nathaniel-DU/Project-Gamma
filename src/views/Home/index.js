import React, { Fragment, useState } from "react";
import { Link } from 'react-router-dom';
import Header from '../../components/Header';
import Button from '../../components/Button/';
import Logo from '../../assets/logo.png'
import './style.css';


const Home = () => {
  return (
      <Fragment>
        <Header text='Welcome to StaySafe!' />
        <img className='staysafe-logo' alt='StaySafe Logo' src={Logo}></img>
        <Link to="/auth/login"><button className='mint-button'>Login</button></Link>
        <Link to="/auth/create"><button className='mint-button'>Sign Up</button></Link>
      </Fragment>
    )
};

export default Home;