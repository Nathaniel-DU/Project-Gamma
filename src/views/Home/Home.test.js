import React from 'react';
import { shallow } from 'enzyme';
import Home from './index';

describe(`Home`, () => {
    it(`should render correctly in "debug" mode`, () => {
        const component = shallow(<Home debug />);
        expect(component).toMatchSnapshot();
    });
});