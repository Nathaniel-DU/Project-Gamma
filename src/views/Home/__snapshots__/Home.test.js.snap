// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly in "debug" mode 1`] = `
<Fragment>
  <Header
    text="Welcome to StaySafe!"
  />
  <img
    alt="StaySafe Logo"
    className="staysafe-logo"
    src="logo.png"
  />
  <Link
    to="/auth/login"
  >
    <button
      className="mint-button"
    >
      Login
    </button>
  </Link>
  <Link
    to="/auth/create"
  >
    <button
      className="mint-button"
    >
      Sign Up
    </button>
  </Link>
</Fragment>
`;
